ClearKeep/Scenes/Home/Modifiers/InCallModifier.swift

```diff
@@ -0,0 +1,284 @@
+//
+//  InCallModifier.swift
+//  ClearKeep
+//
+//  Created by HOANDHTB on 19/07/2022.
+//
+
+import UIKit
+import SwiftUI
+import CommonUI
+import ChatSecure
+var callController: UIViewController?
+
+struct InCallModifier: ViewModifier {
+	@State var isInMinimizeMode: Bool = false
+	
+	@Environment(\.injected) private var injected: DIContainer
+	@Environment(\.presentationMode) private var presentationMode: Binding<PresentationMode>
+	@Binding var isInCall: Bool
+	@ObservedObject var callViewModel: CallViewModel
+	@State private var location: CGPoint = Position.bottomTrailling.location
+	@GestureState private var startLocation: CGPoint? = nil // 1
+	
+	var simpleDrag: some Gesture {
+		DragGesture()
+			.onChanged { value in
+				var newLocation = startLocation ?? location // 3
+				newLocation.x += value.translation.width
+				newLocation.y += value.translation.height
+				self.location = newLocation
+			}.updating($startLocation) { (_, startLocation, _) in
+				startLocation = startLocation ?? location // 2
+			}
+			.onEnded({ _ in
+				self.location = self.getPosition(position: self.location).location
+			})
+		
+	}
+	
+	func body(content: Content) -> some View {
+		ZStack(alignment: .topLeading) {
+			VStack {
+				if isInCall {
+					Spacer()
+				}
+				content
+			}
+			if isInCall {
+				HStack(alignment: .bottom) {
+					if !callViewModel.callGroup {
+						AvatarDefault(.constant(callViewModel.getUserName()), imageUrl: callViewModel.getAvatar())
+							.frame(width: 34, height: 34)
+							.padding(.trailing, 16)
+							.padding(.leading, 24)
+					} else {
+						VStack{
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Home/Modifiers/InCallModifier.swift#L56"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>InCallModifier.swift:56</code></td>
    </tr>
  </tbody>
</table>